lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/stores/postgres/AbstractPostgresKgStore.scala

```diff
@@ -11,147 +11,147 @@ abstract class AbstractPostgresKgStore(protected val dbConfigProvider: DatabaseC
   import profile.api._
 
   protected val SentencesDelimChar = '|'
-  protected val SentencesDelimString = SentencesDelimChar.toString
-
-  lazy val kgEdges = TableQuery[KgEdgeTable]
-  lazy val kgEdgeLabels = TableQuery[KgEdgeLabelTable]
-  lazy val kgEdgeKgSources = TableQuery[KgEdgeKgSourceTable]
-  lazy val kgNodes = TableQuery[KgNodeTable]
-  lazy val kgNodeLabels = TableQuery[KgNodeLabelTable]
-  lazy val kgNodeLabelEdges = TableQuery[KgNodeLabelEdgeTable]
-  lazy val kgNodeLabelEdgeKgSources = TableQuery[KgNodeLabelEdgeKgSourceTable]
-  lazy val kgNodeLabelKgSource = TableQuery[KgNodeLabelKgSourceTable]
-  lazy val kgNodeKgSources = TableQuery[KgNodeKgSourceTable]
-  lazy val kgNodeKgNodeLabels = TableQuery[KgNodeKgNodeLabelTable]
-  lazy val kgSources = TableQuery[KgSourceTable]
-
-  protected def runTransaction[R](a: DBIOAction[R, NoStream, Nothing]): Future[R] = {
+  protected val SentencesDelimString: String = SentencesDelimChar.toString
+
+  protected lazy val edges = TableQuery[EdgeTable]
+  protected lazy val edgeLabels = TableQuery[EdgeLabelTable]
+  protected lazy val edgeSources = TableQuery[EdgeSourceTable]
+  protected lazy val nodes = TableQuery[NodeTable]
+  protected lazy val nodeLabels = TableQuery[NodeLabelTable]
+  protected lazy val nodeLabelEdges = TableQuery[NodeLabelEdgeTable]
+  protected lazy val nodeLabelEdgeSources = TableQuery[NodeLabelEdgeSourceTable]
+  protected lazy val nodeLabelSource = TableQuery[NodeLabelSourceTable]
+  protected lazy val nodeSources = TableQuery[NodeSourceTable]
+  protected lazy val nodeNodeLabels = TableQuery[NodeNodeLabelTable]
+  protected lazy val sources = TableQuery[SourceTable]
+
+  protected final def runTransaction[R](a: DBIOAction[R, NoStream, Effect.All]): Future[R] = {
     db.run(a.transactionally)
   }
 
-  protected def runSyncTransaction[R](a: DBIOAction[R, NoStream, Nothing], duration: Duration = Duration.Inf): R = {
+  protected final def runSyncTransaction[R](a: DBIOAction[R, NoStream, Effect.All], duration: Duration = Duration.Inf): R = {
     Await.result(runTransaction(a), duration)
   }
 
-  protected final case class KgEdgeRow(id: String, objectKgNodeId: String, predicate: String, sentences: String, subjectKgNodeId: String)
-  protected final case class KgNodeRow(id: String, inDegree: Option[Short], outDegree: Option[Short], pageRank: Option[Float], pos: Option[Char], wordNetSenseNumber: Option[Short])
+  protected final case class EdgeRow(id: String, objectNodeId: String, predicate: String, sentences: String, subjectNodeId: String)
+  protected final case class NodeRow(id: String, inDegree: Option[Short], outDegree: Option[Short], pageRank: Option[Float], pos: Option[Char], wordNetSenseNumber: Option[Short])
 
-  private class KgEdgeTable(tag: Tag) extends Table[KgEdgeRow](tag, "edge") {
+  protected class EdgeTable(tag: Tag) extends Table[EdgeRow](tag, "edge") {
```

Comment:
Can make these final, too, for good measure.

Reply:
My habit from Java is to make everything final and then remove it as needed.

Reply:
That makes sense, I will keep that in mind